File: testConnection.py
import pyodbc
import numpy

server = 'DBSED3646.ms.ds.uhc.com'
database = 'SQUAD_DB'
driver= '{ODBC Driver 13 for SQL Server}'
#username = 'your_username'
#password = 'your_password'
cnxn = pyodbc.connect('DRIVER='+driver+';PORT=1433;SERVER='+server+';PORT=1443;DATABASE='+database+';Trusted_Connection=yes;') #;UID='+username+';PWD='+ password#
print('DRIVER='+driver+';PORT=1433;SERVER='+server+';PORT=1443;DATABASE='+database+';Trusted_Connection=yes;')
cursor = cnxn.cursor()

# Testing the ability to fetch data from the database 
cursor.execute("SELECT * FROM Patient_Information")
row = cursor.fetchone()
while row:
    print str(row[0]) + ":" + str(row[1])
    row = cursor.fetchone()

# Testing the script to put data into the database
with cursor.execute("INSERT INTO Patient_Information VALUES ('Daniel', 'Wu', 'M', 1)"):
    print("Successful insertion")

cnxn.commit()

# Testing to see if the function actually returns the data inputed above
cursor.execute("SELECT * FROM Patient_Information")
row = cursor.fetchone()
while row:
    print str(row[0]) + ":" + str(row[1])
    row = cursor.fetchone()

File: testConnection.js
var config = require('./backend/config.js')

var db = require('odbc')(), cn = 'DRIVER=' + config.driver + ';PORT=1433;SERVER='
    + config.server + ';PORT=1443;DATABASE=' + config.database + ';Trusted_Connection=yes';


function testConnection(){
    db.open(cn, function (err) {
        if (err) return console.log(err);
        

        db.query('SELECT * FROM Question', function (err, data) {
            if (err) console.log(err);
        
            console.log(data);
    
            db.close(function () {
                console.log('done');
            });
        });
    });
}
 
module.exports = { 
    testConnection
}